reference.md

```diff
@@ -4,6 +4,57 @@ layout: reference
+   <dt>Digital object identifier (DOI)</dt>
+   <dd><a href="https://glosario.carpentries.org/en/#doi" alt="Link to Carpentries Glossario">See the Carpentries Glossario</a></dd>
+   <dt>Docker</dt>
+   <dd>A software framework for creating, running and managing <em>containers</em>.</dd>
+   <dt>Docker build context</dt>
+   <dd>The docker build command builds Docker images from a Dockerfile and a “context”. A build's context is the set of files located in the specified PATH or URL.</dd>
+   <dt>Docker Hub</dt>
+   <dd>An online library of Docker <em>container images</em>.</dd>
+   <dt>Docker Hub repository</dt>
+   <dd>A collection of related Docker <em>container images</em> hosted on Docker Hub.</dd>
+   <dt>Docker tag</dt>
+   <dd>The specific version identifier associated with a Docker <em>container image</em>.</dd>
+   <dt>Dockerfile</dt>
+   <dd>The file containing the commands to build a Docker <em>container image</em> along with the <em>Docker context</em>.</dd>
+   <dt>Filesystem</dt>
+   <dd><a href="https://glosario.carpentries.org/en/#filesystem" alt="Link to Carpentries Glossario">See the Carpentries Glossario</a></dd>
+   <dt>Filesystem layer</dt>
+   <dd>Each <em>container image</em> is made up of multiple read-only filesystem layers that represent the file system differences from the layers below them in the image.</dd>
+   <dt>Hardware</dt>
+   <dd><a href="https://glosario.carpentries.org/en/#hardware" alt="Link to Carpentries Glossario">See the Carpentries Glossario</a></dd>
+   <dt>Hard drive</dt>
+   <dd>The hardware in a computer that hosts the filesystem (or, sometimes, other storage types).</dd>
+   <dt>Host computer</dt>
+   <dd>The computer system which is running the <em>container</em>.</dd>
+   <dt>Memory/RAM</dt>
+   <dd>Random Access Memory (RAM) is where data the CPU is working with is temporarily stored.</dd>
+   <dt>Operating system (OS)</dt>
+   <dd><a href="https://glosario.carpentries.org/en/#operating_system" alt="Link to Carpentries Glossario">See the Carpentries Glossario</a></dd>
+   <dt>Reproducible research</dt>
+   <dd><a href="https://glosario.carpentries.org/en/#reproducible_research" alt="Link to Carpentries Glossario">See the Carpentries Glossario</a></dd>
+   <dt>Software library</dt>
+   <dd><a href="https://glosario.carpentries.org/en/#library" alt="Link to Carpentries Glossario">See the Carpentries Glossario</a></dd>
+   <dt>Tar archive</dt>
+   <dd>A file archive format commonly used in Unix-like operating systems that combines multiple files into a single file. tar archive files are used as the export format of <em>Docker images</em>.</dd>
+   <dt>Virtualization</dt>
+   <dd><em>Containers</em> are an example of virtualization – having a second “virtual” computer running and accessible from a <em>host computer</em>.</dd>
```

Comment:
I was thinking whether there is a way to explain this in relation to something that provides a virtual, software-based representation of hardware that can support running artefacts designed to run on the "emulated" hardware? I think maybe the description you've provided is more straightforward and I'm perhaps thinking about this in the context of a more technical description which I guess we don't need.

Reply:
I think this is a simple glossary so we do not need "complete" technical descriptions. I lean towards leaving the simple definition for now but if you really feel strongly we can add something more detailed